Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
```

Comment:
Pipfile is a good idea, but I would rather go in the direction of be installable by PIP and have a `podcast-dl` command, so we need a setup.py instead. Pipfile is terrible for integrating with setup.py.
Maybe we could keep it for ease development?